app/controllers/changeContactDetails/OrganisationSecondContactNameController.scala

```diff
@@ -44,7 +45,9 @@ class OrganisationSecondContactNameController @Inject() (
 
   private val form = formProvider()
 
-  def onPageLoad(mode: Mode): Action[AnyContent] = standardActionSets.identifiedUserWithData() {
+  private val allowedAffinityGroups: Set[AffinityGroup] = Set(AffinityGroup.Organisation, AffinityGroup.Agent)
```

Comment:
This `private val` is declared on every one of the organisation controllers. Is it worth declaring it once somewhere and referencing it in all of the controllers?

Or what about if instead of that, we have two methods on the action sets - one to be used by org controllers and one to be used by individual controllers?